docs/en/api/FS-API.md

```diff
@@ -78,6 +78,36 @@ CreateFilePOptions options = CreateFilePOptions.newBuilder().setBlockSizeBytes(6
 FileOutStream out = fs.createFile(path, options);
 ```
 
+#### Programmatically Modifying Configuration
+
+Alluxio configuration can be set through alluxio-site.properties, but these properties apply to all
```

Comment:
maybe use backticks around `alluxio-site.properties` to denote it's a file?

Reply:
Updated, thanks.